src/main/java/se/sundsvall/casemanagement/api/validation/impl/ByggRStakeholderRoleConstraintValidator.java

```diff
@@ -0,0 +1,36 @@
+package se.sundsvall.casemanagement.api.validation.impl;
+
+import java.util.List;
+import java.util.Set;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+import se.sundsvall.casemanagement.api.model.enums.StakeholderRole;
+import se.sundsvall.casemanagement.api.validation.ByggRStakeholderRole;
+
+public class ByggRStakeholderRoleConstraintValidator implements ConstraintValidator<ByggRStakeholderRole, List<String>> {
+
+	@Override
+	public boolean isValid(List<String> roles, ConstraintValidatorContext context) {
+		return roles.stream().allMatch(this::isValidRole);
+	}
+
+	private boolean isValidRole(String role) {
+
+		if (role == null || role.isEmpty()) {
+			return false;
+		}
+
+		// Valid roles
+		final Set<String> validRoles = Set.of(
```

Comment:
This can be moved to be a static list instead of creating it every time method is called.